pkgs/data/fonts/junicode/default.nix

```
@@ -1,20 +1,21 @@
 { lib, fetchzip }:
 
 fetchzip {
-  name = "junicode-0.7.8";
+  name = "junicode-1.002";
 
-  url = mirror://sourceforge/junicode/junicode/junicode-0-7-8/junicode-0-7-8.zip;
+  url = mirror://sourceforge/junicode/junicode/junicode-1.002/junicode-1.002.zip;
```

To enable automatic updates and make this less error-prone, this should make use of the `version`:

```suggestion
  url = mirror://sourceforge/junicode/junicode/junicode-${version}/junicode-${version}.zip;
```